tools/distpartitioning/parmetis_preprocess.py

```diff
@@ -43,6 +44,26 @@ def get_proc_info():
         return 0
 
 
+def get_world_size():
```

Comment:
I wouldn't mind, but then we should move `get_proc_info()` (lines 23-44) as well.

Actually I'm curious why we don't use `torch.distributed.get_world_size()`  and `torch.distributed.get_rank()` for these, they seem to be used in most of DGL distributed.

Reply:
@classicsong can we get a decision here? The options are:

* Move get_world_size() and get_proc_info() to utils.py.
* Remove these functions and use `torch.distributed.get_world_size()` and `torch.distributed.get_rank()` in another PR (as they might need further testing).
* Do nothing, keep both these functions in their current location.

Reply:
move Move get_world_size() and get_proc_info() to utils.py.
As we launch parmetis_preprocess.py using mpirun. So we use mpirun env to get world size.

Reply:
Sounds good. Let's do this in a separate PR to avoid causing trouble with the existing codebase. Once we have better code coverage we can look into refactoring.